test/integration/hooks/validate.test.js

```diff
@@ -108,47 +107,6 @@ describe(Support.getTestDialectTeaser('Hooks'), function() {
         });
       });
     });
-
-    describe('on error', function() {
```

Comment:
That's a really good question. When I initially viewed the tests, they didn't strike me as anything that necessitated being integration tests. In fact, they were written in such a way as to test the functionality of one specific function and had no intrinsic need to use the DB. I figured I would re-write them as unit tests, mostly because the integration tests are slow, and I don't see a reason to include extra ones unnecessarily.

But you're right, now we aren't calling any of the dependencies of `_validateAndRunHooks` anymore. Would you prefer I right unit tests for the rest of the methods of InstanceValidtor?